### DISCUSSION TITLE
Deforum??

### DISCUSSION THREAD

**Discussion by dhmiller** [TheLastBen/fast-stable-diffusion#2635] [General]
Deforum appears in my Extension as as "Latest," but the ver is from May  (or July - 2023-05-07 ). But Deforum doesn't  show up in the UI even after updating and reloading the UI. No doubt because of this error - can anyone help with this? 
*** Error loading script: deforum.py
    Traceback (most recent call last):
      File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/scripts.py", line 382, in load_scripts
        script_module = script_loading.load_module(scriptfile.path)
      File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/script_loading.py", line 10, in load_module
        module_spec.loader.exec_module(module)
      File "<frozen importlib._bootstrap_external>", line 883, in exec_module
      File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
      File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/extensions/deforum-for-automatic1111-webui/scripts/deforum.py", line 41, in <module>
        from webui import wrap_gradio_gpu_call
    ImportError: cannot import name 'wrap_gradio_gpu_call' from 'webui' (/content/gdrive/MyDrive/sd/stable-diffusion-webui/webui.py)

![image](https://github.com/TheLastBen/fast-stable-diffusion/assets/45694140/adda2ce4-2297-465e-90bc-a4d91d928585)
  Thanks for any assistance.

**Comment by TheLastBen** [comment]
try removing the extension and reinstalling it

**Comment by dhmiller** [comment]
Thanks much.
1. Disabled all extensions, then quit. Code shows:
Closing server running on port: 7860 *** "Disable all extensions" option
was set, will not load any extensions *** Running on public URL:
https://2a33a344976ab45819.gradio.live ✔ Connected Startup time: 5.1s
(create ui: 0.7s, gradio launch: 3.5s, add APIs: 0.8s).

2. Restarted AUTO1111 UI at https://2a33a344976ab45819.gradio.live

3. Extensions shows: "Disable all extensions" was set, change it to "none"
to load all extensions again

[image: image.png]

3. "New commits" available as indicated  Deforum
deforum-for-automatic1111-webui
https://github.com/deforum-art/deforum-for-automatic1111-webui.git
automatic1111-webui 9c1300b3
<https://github.com/deforum-art/deforum-for-automatic1111-webui/commit/9c1300b3831b2f79ed429b699386e210c56a076f>
2023-05-07
21:22:04 new commits
but when I choose Check for Updates, Deforum is not listed in the Available
extensions list.
So how would I access the news commits for Deforum?

Thanks again.
Dennis
